my-app/src/pages/AboutUs.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import '../App.css';
+import './AboutUs.css'
+
+export default function AboutUs() {
+  // to create light and dark mode 
+  const [mode, setMode] = useState('light');
+
+  const switchMode = () => {
```

Comment:
you can shorten this function and make it more succinct with a simple ternary operator